client/js/socket-events/init.js

```diff
@@ -31,6 +31,10 @@ socket.on("init", function(data) {
 	$(".show-more button, #input").prop("disabled", false);
 	$("#submit").show();
 
+	if (data.uploads) {
+		$("#upload").show();
```

Comment:
As discussed in IRC, let's keep hide/show in configuration event.

Reply:
Done